docs/stake-transition.md

```diff
@@ -0,0 +1,147 @@
+# Transition of Existing Stakes to the Babylon Chain
+
+This document is intended for Bitcoin stakers who have
+submitted confirmed Bitcoin staking transactions
+on the Bitcoin blockchain before the launch of
+the phase-2 Babylon blockchain (e.g., via participation
+in one of the phase-1 networks).
```

Comment:
```suggestion
in one of the phase-1 caps).
```

Reply:
Networks refers to the testnet, mainnet, devnet etc. not the caps themselves